packaging/build_wheels.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash -eux
+
+#	L I B R A R Y
+
+build_wheel() { (
+	# runs isolated in a sub-shell.
+
+	local runtimedir="$1"
+	local sdist="$2"
+
+	# Activate runtime in a subshell
+	PATH="$runtimedir/bin:$PATH"
+	hash -r
+
+	local runtime="${runtimedir##*/}"
+
+	# Build wheel from source tarball
+	pip --verbose wheel --no-deps --wheel-dir "$distdir" "$sdist"
+	wheel="$distdir/$tarname-$runtime-linux_$AUDITWHEEL_ARCH.whl"
+
+	# Fix wheel to embed libev shared object.
+	auditwheel repair --wheel-dir "$distdir" "$wheel"
+	rm -f "$wheel"
+	wheel="$distdir/$tarname-$runtime-$AUDITWHEEL_PLAT.whl"
+
+	# Pentest wheel in a virtualenv.
+	mkvenv /tmp/bjoern-pentestenv
+	/tmp/bjoern-pentestenv/bin/pip install "$wheel"
+	/tmp/bjoern-pentestenv/bin/python tests/test_wsgi_compliance.py
```

Comment:
Double-check whether you are really testing against installed `bjoern` here. The project doesn't use the `src` layout and you're not changing from the project root dir, so I suspect you will be importing and running code from the repository.

Reply:
Good catch ! I fixed this by moving out from source tree.